edrumulus_parameters.cpp

```diff
@@ -57,6 +57,99 @@ void Edrumulus::Pad::apply_preset_pad_settings()
 
   switch ( pad_settings.pad_type )
   {
+
+    case LEHHS12C: // dual trigger
+      pad_settings.is_rim_switch           = true;
+      pad_settings.scan_time_ms            = 4.0f;
+      pad_settings.decay_fact_db           = 5.0f;
+      pad_settings.decay_len2_ms           = 600.0f;
+      pad_settings.decay_grad_fact2        = 100.0f;
+      pad_settings.rim_shot_is_used        = true;
+      pad_settings.rim_shot_treshold       = 25;
+      pad_settings.rim_shot_boost          = 0;
+      pad_settings.velocity_threshold      = 18;
+      pad_settings.velocity_sensitivity    = 6;
+      break;
+
+    case LEHHS12C_CTRL: // 0-35k poti
+      pad_settings.is_control              = true;
+      pad_settings.pos_threshold           = 6;
+      pad_settings.pos_sensitivity         = 23;
+      pad_settings.velocity_threshold      = 0;
+      pad_settings.velocity_sensitivity    = 31;
```

Comment:
Btw @corrados I'd suggest to increase the range of this parameter as I was already hitting the upper limit here.

Reply:
A sensitivity of 31 is not enough. Hm, that would indicate that you are using just a fraction of the ADC range for your pad. Maybe a result of your custom schematics?

It would be interesting to see what "sqrt(s.peak_val)" you get for your very low volume strikes and loudest possible strikes.

Reply:
It's the hihat control sensitivity which you recently changed.

And IIRC the hihat control is the only place where we both use the exact same schematics with 10k resistors. ;-)

Reply:
Oh, I thought it was a regular pad :-).

Can you please set `pad_settings.velocity_sensitivity = 29` and adjust the factor 1.8 in line https://github.com/corrados/edrumulus/blob/48b29749033af48b4eb13db681dd99038417e0ae/edrumulus.cpp#L563 until you are happy with the results. If you like, you can then also adjust the comment (linear range). Then, please create a separate merge request with the new factor. Thanks.